src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -1052,6 +1048,159 @@ void unpack_u4f16_asymm_zp(const ov::SoPtr<ov::ITensor>& from,
     }
 }
 
+void unpack_u8f16_asymm_zp(const ov::SoPtr<ov::ITensor>& from,
+                           const ov::SoPtr<ov::ITensor>& zerop,
+                           const ov::SoPtr<ov::ITensor>& scale,
+                           const ov::SoPtr<ov::ITensor>& to,
+                           const ov::npuw::util::UnpackOptions& unpack_options) {
+    NPUW_ASSERT(from->is_continuous());
+    NPUW_ASSERT(zerop->is_continuous());
+    NPUW_ASSERT(scale->is_continuous());
+    NPUW_ASSERT(to->is_continuous());
+    NPUW_ASSERT(from->get_size() == to->get_size());
+
+    const auto& from_shape = from->get_shape();
+    NPUW_ASSERT(from_shape.back() % 64 == 0);
+
+    // 2-channel (Symmetric) and 3-channel (group-wise)
```

Comment:
I'm not so sure about this change. What is asymm unpack? Is it always 3 channel parameters?